scripts/submit_eval_jobs.py

```diff
@@ -56,32 +56,27 @@
     "alpaca_eval",
 ]
 
-# format: model name, their beaker id, checkpoint subfolder, tuned or base.
-# or: name, path, None, tuned or base
-models = [
-    # llama1 models
-    # ("llama1-7B", "01HCCBK1MYKXKQC0C6CSVW1F22", None, "vanilla_lm"),
-
-    # other causal models
-    # ("hf-opt-7B", "facebook/opt-6.7b", None, "vanilla_lm"),
-    # ("finetuned_opt", "01H13EBXSADXXJCRERART90ZKJ", None, "tuned_lm"),
-    (args.model_name, args.location, args.beaker_subfolder, model_type),
-]
+model_info = (args.model_name, args.location, args.beaker_subfolder, model_type)
```

Comment:
Could you leave the comments for hf/beaker/etc examples? Just to make it extra clear what these arguments should be.

Reply:
We also lose the ability to batch-submit models (via the list) here, but this is probably fine.